### PR TITLE
Add CAN FD functionality

### PR DESCRIPTION
**Pull request by Acuadros95**
TODO:
- [ ] Define CAN identifier for micro-ROS
- [x] Filter messages for other devices.
- [x] Add to integration XRCE tests.

Related:
- https://github.com/eProsima/Micro-XRCE-DDS/pull/114
- https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/278

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/517/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/518/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/518/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/519/)

File: include/uxr/agent/utils/ArgumentParser.hpp [hunk 1]
```diff
@@ -1097,6 +1161,25 @@ template<> inline bool ArgumentParser<PseudoTerminalAgent>::launch_agent()
 
     return false;
 }
+
+template<> inline bool ArgumentParser<CanAgent>::launch_agent()
+{
+    // TODO: include can id in args?
```
Comment (Acuadros95) [inline include/uxr/agent/utils/ArgumentParser.hpp:1167]:
```suggestion
```

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/519/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/520/)

**Review by pablogs9 — APPROVED** [review]

**Pull request merged**